hs_core/tasks.py

```diff
@@ -461,17 +464,15 @@ def create_new_version_resource_task(ori_res_id, username, new_res_id=None):
         utils.copy_and_create_metadata(ori_res, new_res)
 
         # add or update Relation element to link source and target resources
-        hs_identifier = new_res.metadata.identifiers.all().filter(name="hydroShareIdentifier")[0]
-        ori_res.metadata.create_element('relation', type='isReplacedBy', value=hs_identifier.url)
+        ori_res.metadata.create_element('relation', type='isReplacedBy', value=new_res.get_citation())
```

Comment:
get_citation() returns more than the resource's identifier URL, right? What is the rationale to store resource citation in the value of the relation rather than the resource's identifier?

Reply:
It is the new requirement as specified in #3261 to store the resource citation instead of the resource URL. The resource citation is then reflected in resourcemetadata.xml file for the relation term. It is also a requirement to display resource citation for the relation on the UI.